components/trakt/sources/new-show-or-movie-watched/new-show-or-movie-watched.mjs

```diff
@@ -0,0 +1,46 @@
+import trakt from "../../trakt.app.mjs";
+import constants from "../common/constants.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  name: "New Show Or Movie Watched",
+  version: "0.0.1",
+  key: "trakt-new-show-or-movie-watched",
+  description: "Emit new event on each new watched show or movie.",
+  type: "source",
+  dedupe: "unique",
+  props: {
```

Comment:
some property are shared between other sources, it could be placed in a common/base file